SQL/Archive/17.0/2016-07-08-AddProjectIDTotest_batteryTable.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE test_battery ADD `ProjectID` int(11) DEFAULT NULL;
```

Comment:
Yes, but most importantly BEFORE SubprojectID and... look at this kitten:

![image](https://cloud.githubusercontent.com/assets/2903505/16660780/d4714b64-443d-11e6-962e-a992656e31dc.png)